### DISCUSSION TITLE
Question on query.

### DISCUSSION THREAD

**Discussion by rinuthomashm** [bokwoon95/go-structured-query#15] [Q&A] [answered]
Hallo team, Good day to you.
Could anyone please help generating a query like this?

update table1 set row1 = row1 + 15 where id = 25.

**Comment by bokwoon95** [comment, marked as answer]
Hi, it looks something like this:

```go
q := sq.
    Update(tbl).
    Set(
        tbl.ROW1.Set(sq.Fieldf("? + ?", tbl.ROW1, 15)),
    ).
    Where(tbl.ID.EqInt(25))
```

Hope that helps.

**Reply by rinuthomashm** [reply]
Thank you very much it worked.
